torchchat/usages/openai_api.py

```diff
@@ -17,11 +17,10 @@
 
 import torch
 
-from torchtune.models.llama3_2_vision._convert_weights import padded_collate
-from torchtune.models.llama3_2_vision._model_builders import llama3_2_vision_transform 
-
 from PIL import Image
-from torchtune.data import Message
+
+from torchtune.data import Message, padded_collate
```

Comment:
we need to double check about `padded_collate` func here;  in the generation.py we've replaced it with `padded_collate_tiled_images_and_mask`